cafebabel/static/css/styles.css

```diff
@@ -903,10 +915,26 @@ button.share-button.active:after {
 .create-article-page .article-settings {
     max-width: 75rem;
     margin: auto;
+    text-align: justify;
 }
 
-.create-article-page .select {
-    display: inline-block;
+.create-article-page .article-settings:after { /* https://codepen.io/gcyrillus/pen/ruhCg */
+    content: '';
+    padding-right: 100%;
```

Comment:
does this work without a `display: block (or inline-block)`?

Reply:
no